website/project/model.py

```diff
@@ -1587,6 +1594,10 @@ def save(self, *args, **kwargs):
         if settings.PIWIK_HOST and update_piwik:
             piwik_tasks.update_node(self._id, saved_fields)
 
+        # Ensure Mailing list subscription is set up
+        if first_save and self.mailing_enabled:
+            auth_signals.node_created.send(self)
```

Comment:
Okay, since we've got the `mailing_enabled` check in the signal function, let's remove it here and otherwise keep the signal as-is.